web/ci.sh

```diff
@@ -43,18 +43,14 @@ TIER=`cat ../TIER.md`
 BUILD_NUMBER=`cat ../VERSION.md`
 
 function web_sentry_upload () {
-  if [ $1 = "webview" ]; then
-    # There is no "publish" version for app/webview; it's "published" inside our mobile apps.
-    ARTIFACT_FOLDER="$KEYMAN_ROOT/web/build/app/webview/release/"
-  elif [ $1 = "browser" ]; then
-    ARTIFACT_FOLDER="$KEYMAN_ROOT/web/build/publish/release/"
-  fi
+  echo "Uploading $1 to Sentry..."
 
-  pushd "$ARTIFACT_FOLDER"
-  echo "Uploading to Sentry..."
-  sentry-cli releases files "$VERSION_GIT_TAG" upload-sourcemaps --strip-common-prefix "@keymanapp/keyman/" --rewrite --ext js --ext map --ext ts || fail "Sentry upload failed."
+  # The "$ARTIFACT_FOLDER" bit is being used as the path to the sourcemaps.
```

Comment:
This comment doesn't make sense with the changes -- there is no `$ARTIFACT_FOLDER` in the remaining code

Reply:
```suggestion
```